src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_panel.test.tsx

```diff
@@ -124,12 +132,20 @@ describe('WorkspacePermissionSettingInput', () => {
     });
   });
   it('should call onChange with new group type', () => {
+    Object.defineProperty(HTMLElement.prototype, 'offsetHeight', {
```

Comment:
Shall we move these offset mocks to the beforeEach / afterEach? I think we should clear it after test case complete.

Reply:
ok